lib/src/firebase/dbRefs.dart

```diff
@@ -4,4 +4,16 @@ class DbRefs {
   Firestore _fs;
 
   DbRefs(this._fs);
+
+  userRef(){
+    _fs.collection('users');
+  }
```

Comment:
This reference is more or less going to be getting "all users", and since we will end up needing a ref to a single user too, it might be best to name this something like `allUsers`. Also, since this is namespaced to a class called dbRefs, I don't think we really need the word `ref` in there unless you like it a lot.

Also, you'll want to return the output of `_fs.collection('users');`, and also add a documentation comment to the function, just so we can easily remember what all of this stuff does in a month. So all of that together would look like this, if you are reffing a single user:
``` dart
/// [user] takes a unique identifier for a user, and returns a DocumentReference to a specific user
user(String uid) => _fs.collection('users').doc(uid);
```

Reply:
The allusers one would look kinda like this, in theory

``` dart
/// [allUsers] returns a CollectionReference to the collection of all users
allUsers() => _fs.collection('users');
```